recommenders/popularity_recommender.py

```diff
@@ -1,31 +1,27 @@
-from recommenders import data_movies, data_ratings, default_poster_url
-import pandas as pd
-
-
-def recommend_popular_movies():
-
-    ######################################################################################################
-    movie_ratings = None
```

Comment:
The variable 'movie_ratings' is declared as None but is later reassigned. It is not necessary to declare it as None at the beginning.